src/odemis/gui/util/wx_adapter.py

```diff
@@ -21,6 +21,7 @@
 
 import wx
 import sys
+import io
```

Comment:
The change is fine, but the commit message is quite confusing. I'd suggest something like:
```
gui util wx_adapter: Fix loading packaged images with wxPython3

The wxPython3 wrapper only converted from wx.Image() to wx.ImageFromStream() when the argument is a file.
On a Linux installation, that's the case. However, on Windows, Odemis is packaged, and so pkg_resource doesn't provide actual files, but ByteIO. In order to handle them too, we need to explicitly check for this argument type as well.
Note that an alternative could have been to do what wxPython4 seems to check for a "Stream": whether the object has "read" method or not.
```